### PR TITLE
reload default CNI network name real-time if it is not assigned in initialization

### PR DESCRIPTION
**Pull request by mars1024**
ref to #31 

I think the preconditions (del from cache) have been met in #56 , so we can go on promoting this, and for clarity, I pull a new request...

@dcbw @haircommander @mrunalp 

Signed-off-by: Bruce Ma <brucema19901024@gmail.com>

### CODE REVIEW

**Comment by mars1024** [comment]
@dcbw @mrunalp @haircommander Could you help to review this PR? Thanks

**Comment by squeed** [comment]
I think this is fine for now... Ideally, we should refactor this function so that it doesn't try and load all networks all the time.

But for now, this is a fine change.

File: pkg/ocicni/ocicni.go [hunk 1]
```diff
@@ -186,6 +191,8 @@ func (plugin *cniNetworkPlugin) monitorConfDir(start *sync.WaitGroup) {
 // If defaultNetName is not empty, a CNI config with that network name will
 // be used as the default CNI network, and container network operations will
 // fail until that network config is present and valid.
+// If defaultNetName is empty, CNI config files should be reloaded real-time and
```
Comment (dcbw) [inline pkg/ocicni/ocicni.go]: "should be" -> "will be"

**Pull request merged**